scripts/simplify_network.py

```diff
@@ -129,6 +129,11 @@ def simplify_network_to_base_voltage(n, linetype, base_voltage):
     # Note: s_nom is set in base_network
     n.lines["num_parallel"] = n.lines.eval("s_nom / (sqrt(3) * v_nom * i_nom)")
 
+    # Re-define s_nom for DC lines
+    n.lines.loc[n.lines["carrier"] == "DC", "num_parallel"] = n.lines.eval(
+        "s_nom / (v_nom * i_nom)"
+    )
```

Comment:
Super! :) we are there!

I'd recommend to create an auxiliary variable `is_dc_carrier = n.lines["carrier"] == "DC"` and delimit the right-hand side to avoid potential problems
`n.lines.loc[is_dc_carrier , "num_parallel"] = n.lines.loc[is_dc_carrier].eval("s_nom / (v_nom * i_nom)")`

I believe this should work but we have to rely on the fact that the indices that do not match are dropped; it may be better to stay safe.
To be merged after this :)

Reply:
A great point! Fixed